Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -799,7 +799,7 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
             )
         } catch {
             if isSubscriptionEnabled, let error = error as? NetworkProtectionError, case .vpnAccessRevoked = error {
-                await handleInvalidEntitlement(attemptsShutdown: false)
+                await handleInvalidEntitlement(attemptsShutdown: true)
```

Comment:
Initially I thought there's no need to attempt a shutdown; but this is necessary when the app tries to automatically reconnect for whatever reason.